packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/store/init.ts

```diff
@@ -231,12 +232,24 @@ export function initEditorState(
   // Mark all nodes that should be in 'request' phase
   markRequestNodes(['request', 'service_request'])
 
+  let cacheConfig: CacheConfigFormData | null = null
+  if (pluginData.config?.resources?.cache) {
+    cacheConfig = {
+      ...pluginData.config.resources.cache,
+    }
+
+    if (pluginData.partials) {
+      cacheConfig.partial_id = pluginData.partials[0].id
```

Comment:
I think we should check the type of the partial, at lease we should ensure it's not some unknown value (may be added in the future).